docs/release/release-tasks.md

```diff
@@ -0,0 +1,187 @@
+# Release Tasks
+
+**Notes**:
+
+- The examples in this document are based on the v1.8 release cycle.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Prepare main branch for development of the new release](#prepare-main-branch-for-development-of-the-new-release)
+- [Remove previously deprecated code](#remove-previously-deprecated-code)
+- [[Optional] Bump the Kubernetes version](#optional-bump-the-kubernetes-version)
+- [Bump dependencies](#bump-dependencies)
+- [Create a release branch](#create-a-release-branch)
+- [Cut a release](#cut-a-release)
+- [[Continuously] Reduce the amount of flaky tests](#continuously-reduce-the-amount-of-flaky-tests)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Prepare main branch for development of the new release
+
+TODO(sbueringer): Finalize this section once we do this for the first time
+
```

Comment:
I would continue either tomorrow or on Monday. Basically we didn't do a lot of this stuff yet for v1.8

But this is also because we don't have this folder structure yet

Reply:
I meant I would continue with the other PR. I would merge this one here as is and then update this section in the other PR :)

Reply:
I think this section doesn't have to be perfect yet. The main goal is to get this doc in place :)